File: MyHeart.Web/ClientApp/models/professionInfo/SymptomDto.ts
import SymptomDiseaseDto from "./bonds/SymptomDiseaseDto";
import SymptomQuestionsSymptomDto from "./bonds/SymptomQuestionsSymptomDto";
import SymptomSynonymDto from "./SymptomSynonymDto";

export default class SymptomDto {
    id: number;
    name: string | null;
    description: string | null;
    diseases: SymptomDiseaseDto[];
    symptomQuestions: SymptomQuestionsSymptomDto[];
    synonyms: SymptomSynonymDto[];
}

File: MyHeart.DTO/ProfessionInformation/SymptomDTO.cs
using MyHeart.DTO.ProfessionInformation;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace MyHeart.DTO.ProfessionalInformation
{
    public class SymptomDTO : BaseEntityDTO
    {
        [Required]
        public string Name { get; set; }
        public string Description { get; set; }

        public List<Disease_Symptoms_SymptomsDTO> Diseases { get; set; }

        public ICollection<SymptomQuestions_SymptomDTO> SymptomQuestions { get; set; }
        public ICollection<Symptom_SynonymsDTO> Synonyms { get; set; }

    }
}
